dev-ostelco-ios-clientTests/CoordinatorTests/EdgeCasesInStageDeciderTests.swift

```diff
@@ -17,22 +17,16 @@ class EdgeCasesInStageDeciderTests: XCTestCase {
     func testUserSignsUpOnNewDeviceAfterCompletingOnboardingOnOtherDevice() {
         let decider = StageDecider()
         let localContext = LocalContext(hasFirebaseToken: true)
-        
-        let context = Context(
-            customer: CustomerModel(id: "xxx", name: "xxx", email: "xxxx@gmail.com", analyticsId: "xxxx", referralId: "xxxx"),
-            regions: [
-                RegionResponse(
-                    region: Region(id: "sg", name: "Singapore"),
-                    status: .APPROVED,
-                    simProfiles: [
-                        SimProfile(eSimActivationCode: "xxx", alias: "xxx", iccId: "xxx", status: .INSTALLED)
-                    ],
-                    kycStatusMap: KYCStatusMap(jumio: .PENDING, myInfo: .APPROVED, nricFin: .PENDING, addressPhone: .PENDING)
-                )
-            ]
+        let region = RegionResponse(
+            region: Region(id: "sg", name: "Singapore"),
+            status: .APPROVED,
+            simProfiles: [
+                SimProfile(eSimActivationCode: "xxx", alias: "xxx", iccId: "xxx", status: .INSTALLED)
```

Comment:
What does the "xxx" mean? Please add explanatory comment.

Reply:
It means the value doesn't matter. This is used all over the tests in places where the exact value doesn't matter. Not going to add a comment for all such values.